src/ansi-c/library/string.c

```diff
@@ -462,7 +462,7 @@ inline int strncmp(const char *s1, const char *s2, size_t n)
                          "strncmp zero-termination of 2nd argument");
   #else
   __CPROVER_size_t i=0;
-  unsigned char ch1, ch2;
+  char ch1, ch2;
```

Comment:
Can you please also apply the fix to `strncasecmp`?

Reply:
And `strcmp` and `strcasecmp`?

Reply:
Done.